app/src/androidTest/java/com/github/polypoly/app/commons/MockDB.kt

```diff
@@ -19,9 +20,9 @@ class MockDB: IRemoteStorage {
     override fun <T : Any> getValue(key: String, clazz: KClass<T>): CompletableFuture<T> {
         val keyCleaned = cleanKey(key)
         if (!data.containsKey(keyCleaned)) {
-            val failure = CompletableFuture<T>()
-            failure.completeExceptionally(IllegalAccessException("Invalid key $keyCleaned"))
-            return failure
+            val failedPromise = CompletableFuture<T>()
+            failedPromise.completeExceptionally(IllegalAccessException("Invalid key $keyCleaned"))
```

Comment:
`NoSuchElementException`? IllegalAccess is for reflection (I realize this is existing code)